Sources/ScipioKit/DescriptionPackage.swift

```diff
@@ -267,6 +267,11 @@ struct BuildProduct: Hashable, Sendable {
         target.underlying as? ScipioBinaryModule
     }
 
+    static func == (lhs: Self, rhs: Self) -> Bool {
+        lhs.target.name == rhs.target.name &&
+        lhs.package.identity == rhs.package.identity
+    }
```

Comment:
@dmhts You introduced the new `hash` method on #132.
However, I found there are some cases where models are not equal even if the hashable value is the same.
It causes `Set` can't work correctly even if `hashValue` s are identical.

So we need this override.

Reply:
Ah, thank you, I should've implemented it in my PR! Absolutely, I bet the default synthesized conformance to `Equatable` also uses the newly introduced build triple (`BuildTriple`), which might be different for the same target depending on its parent (as I described in the comment a bit below). It precisely explains why two objects with the same hash still might not be considered equal.